packages/shared/src/modules/points-of-interest/libs/validation-schemas/point-of-interest-create.validation-schema.ts

```diff
@@ -0,0 +1,21 @@
+import { z, type ZodType } from "zod";
+
+import {
+	PointsOfInterestValidationMessage,
+	PointsOfInterestValidationRule,
+} from "../enums/enums.js";
+import { type PointsOfInterestRequestDto } from "../types/types.js";
+
+const pointOfInterestCreate: ZodType<PointsOfInterestRequestDto> =
```

Comment:
the type should be inferred from zod library, and there is no need to type it here
```suggestion
const pointOfInterestCreate
```